storage/innobase/xtrabackup/doc/source/xtrabackup_bin/partial_backups.rst

```diff
@@ -37,23 +37,32 @@ named ``test`` which contains tables named ``t1`` and ``t2``.
 Creating Partial Backups
 ================================================================================
 
-There are two ways of specifying which part of the whole data will be backed up:
-enumerating the tables in a file (:option:`--tables-file`) or providing a list
-of databases (:option:`--databases`).
+There are multiple ways of specifying which part of the whole data is backed up:
```

Comment:
This does not mack sense to me 
there are 2 '--tables-file' with different descriptions 

the 2 '--database...' entries have the same description

What is the intent and usage of the options?

Reply:
Fixed.
The options provide a method of selecting either tables or databases to be backed up.  The tables option and the database option require you to list the selected objects in the command line. The tables-file option and the database-file option allow you to have a list of items in a file and to use that file when performing a backup.